pvnet/models/multimodal/multimodal.py

```diff
@@ -289,6 +301,13 @@ def forward(self, x):
         if self.adapt_batches:
             x = self._adapt_batch(x)
 
+        if self.use_id_embedding:
+            id = torch.tensor(
```

Comment:
could you an a comment with an example, like `x['gsp_id] = 1` with location_id_mapping  = {1:0}, would give [0], 
or something like that, just so its slightly easier to read and know what soging on